overdrive.py

```diff
@@ -407,6 +406,7 @@ def _do_post(self, url, payload, headers, **kwargs):
 class MockOverdriveAPI(OverdriveAPI):
 
     def __init__(self, _db, collection=None, *args, **kwargs):
+        self.access_token_requests = []
```

Comment:
In https://github.com/NYPL-Simplified/circulation/pull/473 we become very interested in which requests were made to `token_post`, because we need to see whether we included a PIN or told Overdrive not to check the PIN.